tools/ReleaseTool/GitHubClient.cs

```diff
@@ -35,10 +35,7 @@ public GitHubClient(IGitHubOptions options)
             
         public void LoadCredentials()
         {
-            using (var reader = new StreamReader(Common.ReplaceHomePath(options.GitHubTokenFile)))
-            {
-                octoClient.Credentials = new Credentials(reader.ReadLine());
-            }
+            octoClient.Credentials = new Credentials(File.ReadAllText(Common.ReplaceHomePath(options.GitHubTokenFile)));
```

Comment:
Do you need to `ReplaceHomePath` here? Surely C#'s standard library can resolve `~` in a path?

Reply:
It couldn't but I will dig deeper for a solution.

Either way this is only used to get the GitHub API key (which will be removed when we add a better secret sharing flow).